app/models/folio/request.rb

```diff
@@ -26,7 +27,12 @@ def key
     end
 
     def patron_key
-      record.dig('details', 'userId')
+      record.dig('details', 'proxyUserId') || @patron_id
```

Comment:
```suggestion
      record.dig('details', 'proxyUserId') || record.dig('details', 'userId')
```

Do we need to pass through the original patron? I thought the FOLIo request would have that information somewhere already.

Reply:
I kept getting null for non proxies. I was confused about this. Doing this made it work but perhaps too heavy a solution / not the right fix.

Reply:
At the very least, there is no such thing as `details, userId` in a request
![Screenshot 2023-07-12 at 16 40 35](https://github.com/sul-dlss/mylibrary/assets/1328900/2481e0f2-6d31-409e-93e3-3567045d0d26)

Reply:
`requesterId`, maybe?

Reply:
Oof.